activitysim/cli/benchmark.py

```diff
@@ -0,0 +1,255 @@
+import os
+import sys
+import json
+import shutil
+
+ASV_CONFIG = {
+    # The version of the config file format.  Do not change, unless
+    # you know what you are doing.
+    "version": 1,
+    # The name of the project being benchmarked
+    "project": "activitysim",
+    # The project's homepage
+    "project_url": "https://activitysim.github.io/",
+    # The URL or local path of the source code repository for the
+    # project being benchmarked
+    "repo": ".",
+    # The tool to use to create environments.
+    "environment_type": "conda",
+    # the base URL to show a commit for the project.
+    "show_commit_url": "http://github.com/ActivitySim/activitysim/commit/",
+    # The Pythons you'd like to test against.  If not provided, defaults
+    # to the current version of Python used to run `asv`.
+    # "pythons": ["2.7", "3.6"],
+    # The list of conda channel names to be searched for benchmark
+    # dependency packages in the specified order
+    "conda_channels": ["conda-forge"],
+    # The matrix of dependencies to test.  Each key is the name of a
+    # package (in PyPI) and the values are version numbers.  An empty
+    # list or empty string indicates to just test against the default
+    # (latest) version. null indicates that the package is to not be
+    # installed. If the package to be tested is only available from
+    # PyPi, and the 'environment_type' is conda, then you can preface
+    # the package name by 'pip+', and the package will be installed via
+    # pip (with all the conda available packages installed first,
+    # followed by the pip installed packages).
+    "matrix": {
+        "pyarrow": [],
+        "numpy": [],
+        "openmatrix": [],
+        "pandas": [],
+        "pyyaml": [],
+        "pytables": [],
+        "toolz": [],
+        "orca": [],
+        "psutil": [],
+        "requests": [],
+        "numba": [],
+        "coverage": [],
+        "pytest": [],
+        "cytoolz": [],
+    },
+    # The directory (relative to the current directory) to cache the Python
+    # environments in.  If not provided, defaults to "env"
+    # "env_dir": "../activitysim-asv/env",
+    # The directory (relative to the current directory) that raw benchmark
+    # results are stored in.  If not provided, defaults to "results".
+    # "results_dir": "../activitysim-asv/results",
+    # The directory (relative to the current directory) that the html tree
+    # should be written to.  If not provided, defaults to "html".
+    # "html_dir": "../activitysim-asv/html",
+    # List of branches to benchmark. If not provided, defaults to "master"
+    # (for git) or "default" (for mercurial).
+    "branches": ["master", "develop", "performanceTest"],
```

Comment:
Does it make sense to have hard coded branches here?

Reply:
Not at this point.  I've added a commit to change this to get the current branch dynamically.